packages/frontend/src/components/NavBar/ExploreMenu.tsx

```diff
@@ -117,6 +117,18 @@ const ExploreMenu: FC<Props> = memo(({ projectUuid }) => {
                                 title="Query using SQL runner"
                                 description="Access your database to run ad-hoc queries."
                                 to={`/projects/${projectUuid}/sql-runner`}
+                                onClick={(event) => {
+                                    if (
+                                        history.location.pathname ===
+                                        `/projects/${projectUuid}/sql-runner`
```

Comment:
if I'm editing an existing chart, my pathname is different (https://lightdash-pr-12180.onrender.com/projects/3675b69e-8324-4110-bdca-059031aa8da3/sql-runner/saved-sql-chart/edit) I think we should also open these URLs in a different path. 

Perhaps we can see if pathname startsWith `/projects/${projectUuid}/sql-runner`

apart from that, it works well.

Reply:
Good point! Updated